src/bots/comp/constants.js

```diff
@@ -0,0 +1,70 @@
+const compConstants = {
+  color: '',
+  compensationFields: [
+    {
+      example: 'Mozilla',
+      name: 'Company',
+    },
+    {
+      example: 'Software Engineer I',
+      name: 'Title',
+    },
+    {
+      example: 'software Engineer',
+      name: 'Date of Offer',
+    },
+    {
+      example: 'Mountain View, California',
+      name: 'Location',
+    },
+    {
+      example: '$70,000',
+      name: 'Salary',
+    },
+    {
+      example: 'None',
+      name: 'Stock/Recurring Bonus',
+    },
+    {
+      example: '$5,000',
+      name: 'Relocation/Signing Bonus',
+    },
+    {
+      example: '80k/year',
+      name: 'Total Compensation',
+    },
+  ],
+  compensationFormatter: function (fields, isExample) {
+    let str = '';
+    if(!isExample) str += '```';
+    str += '\n';
+    str += '++salary';
+    str += '\n';
+    fields.forEach(field => {
+      str += field.name + ': ';
+      if (isExample) str += field.example;
+      str += '\n';
+    });
+    if(!isExample) str += '```';
```

Comment:
i think we should just get rid of these ticks in general. it'll look pretty enough without